### PR TITLE
Hide redirectStart and redirectEnd for navigations with cross-origin redirects

### PR DESCRIPTION
**Pull request by noamr**
Closes https://github.com/whatwg/html/issues/7104

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/navigation-timing/pull/170.html" title="Last updated on Feb 24, 2022, 9:29 AM UTC (50f595c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/navigation-timing/170/0f2807d...50f595c.html" title="Last updated on Feb 24, 2022, 9:29 AM UTC (50f595c)">Diff</a>

### CODE REVIEW

**Comment by noamr** [comment]
@yoavweiss I think we can also do this without changing the HTML spec, by hiding `redirectStart` and `redirectEnd` if `redirectCount` is 0. Does that sound OK or too hacky/implicit?

**Comment by yoavweiss** [comment]
I think that's reasonable.

**Comment by noamr** [comment]
> I think that's reasonable.

Done

File: index.html [hunk 1]
```diff
@@ -283,6 +283,24 @@ <h3>
...
+            </ol>
+          </li>
+          <li>The {{PerformanceResourceTiming/redirectEnd}} getter steps are to perform the following steps:
+            <ol>
+              <li>If |this|'s [=PerformanceNavigationTiming/redirect count] is 0, return 0.
+                <li>Otherwise return |this|'s {{PerformanceResourceTiming/redirectEnd}}.
+            </ol>
+            <p class="note">
+              Though `redirectStart` and `redirectEnd` are exposed in <a data-cite=
+              "RESOURCE-TIMING-2#dom-performanceresourcetiming">PerformanceResourceTiming</a>, it
```
Comment (yoavweiss) [inline index.html:299]: s/it has/they have/

File: index.html [hunk 2]
```diff
@@ -283,6 +283,24 @@ <h3>
...
+              has a different meaning in Navigation Timing, which returns zero for navigations with
```
Comment (yoavweiss) [inline index.html:300]: s/which returns/where they return/

**Review by yoavweiss — APPROVED** [review]
LGTM % nits

**Pull request merged**